src/modules/localization/src/virtual_box_marker/virtual_box_marker.py

```diff
@@ -73,15 +64,10 @@ def create_virtual_box_base_link(self):
                                          Point32(vbox_x_upper_bound, vbox_y_lower_bound, vbox_z_upper_bound),
                                          Point32(vbox_x_upper_bound, vbox_y_lower_bound, vbox_z_lower_bound)]
 
-        # polygonFrontMsg.header.frame_id = "base_link"
-        # polygonRearMsg.header.frame_id = "base_link"
-        # polygonRightMsg.header.frame_id = "base_link"
-        # polygonLeftMsg.header.frame_id = "base_link"
-
-        polygonFrontMsg.header.frame_id = "velodyne"
-        polygonRearMsg.header.frame_id = "velodyne"
-        polygonRightMsg.header.frame_id = "velodyne"
-        polygonLeftMsg.header.frame_id = "velodyne"
+        polygonFrontMsg.header.frame_id = "base_link"
```

Comment:
Can the Velodyne point cloud be visualized in the frame?

Reply:
I'd have to create a new issue to define the `tf` from `velodyne` to `base_link`

Reply:
It was being addressed in the `google_cartographer` issue, but that fell apart so I'll just make a new issue for it

Reply:
sounds like a plan. will that new issue have to be merged before this one then? if so, maybe note that somewhere

Reply:
Nah this pull request can proceed, I can finish up the new issue (#243) some time tomorrow